### DISCUSSION TITLE
Healthcheck failed Tesla Model S

### DISCUSSION THREAD

**Discussion by cptmellow** [teslamate-org/teslamate#1242] [Q&A]
Hi

I am not able to get my car to communicate to the app. It was working yesterday but it doesn't connect anymore. I am seeing the token refresh fail from the logs.  Log snippet below. Are you able to help please?

05:04:05.974 [warn] Token refresh failed: %TeslaApi.Error{
teslamate_1  |   env: %Mint.TransportError{reason: :nxdomain},
teslamate_1  |   message: "An unknown error has occurred: :nxdomain",
teslamate_1  |   reason: :unknown

**Comment by cptmellow** [comment]
I also did some DNS resolutions and they seem to working fine. 

; <<>> DiG 9.11.5-P4-5.1+deb10u2-Raspbian <<>> owner-api.teslamotors.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 33017
;; flags: qr rd ra; QUERY: 1, ANSWER: 3, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;owner-api.teslamotors.com.     IN      A

;; ANSWER SECTION:
owner-api.teslamotors.com. 43884 IN     CNAME   owner-api.vn.tesla.services.
owner-api.vn.tesla.services. 4  IN      CNAME   qts-ingress-prd.vn.tesla.services.
qts-ingress-prd.vn.tesla.services. 24 IN A      205.234.31.120

;; Query time: 18 msec
;; SERVER: 75.75.75.75#53(75.75.75.75)
;; WHEN: Fri Jan 01 22:18:54 GMT 2021
;; MSG SIZE  rcvd: 141

**Comment by gary-hammer** [comment]
That only shows the DNS is responding.
Try this to see if the server is live at a different location.

https://www.isitdownrightnow.com/owner-api.teslamotors.com.html
